rust-api/src/main.rs

```diff
@@ -0,0 +1,49 @@
+extern crate dotenv;
+
+use color_eyre::eyre::Result;
+use color_eyre::Report;
+use dotenv::dotenv;
+use poem::middleware::Cors;
+use poem::EndpointExt;
+use poem::{listener::TcpListener, Route, Server};
+use poem_openapi::OpenApiService;
+use sqlx::postgres::PgPool;
+use tracing_subscriber::EnvFilter;
+
+mod api;
+mod forms;
+
+fn setup() -> Result<(), Report> {
+    dotenv().ok();
+    color_eyre::install()?;
+    tracing_subscriber::fmt::fmt()
+        .with_env_filter(EnvFilter::from_default_env())
+        .init();
+
+    Ok(())
+}
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    setup()?;
+
+    let pool = PgPool::connect("postgres://postgres:password@localhost/postgres").await?;
+
+    let api_service = OpenApiService::new(
+        api::Api, 
+        "Forms API Rust Prototype", 
+        env!("CARGO_PKG_VERSION"))
```

Comment:
@james-sheppard-gov-uk being a genius suggsted we get the semver from the cargo.toml :ok_hand: